src/hooks/index.ts

```diff
@@ -1,3 +1,5 @@
+export * from "./amm";
+
 export {
```

Comment:
should we use * for the rest of the export? or is it better to specify the functions/types we want to export

Reply:
Explicit imports